src/interactive-window/interactiveWindow.ts

```diff
@@ -519,7 +522,7 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
         const promises = cells.map((c) => {
             // Add the cell first. We don't need to wait for this part as we want to add them
             // as quickly as possible
-            const notebookCellPromise = this.addNotebookCell(c, fileUri, line, isDebug);
+            const notebookCellPromise = this.addNotebookCell(c, fileUri, line);
```

Comment:
We cannot generate the code for IW cell at this stage, it can only be done once we have resolved the kernel.
E.g. if using remote kernel, then we don't inject `debugger()` statement.